pkg/operator/ceph/cluster/mgr/mgr_test.go

```diff
@@ -66,8 +66,12 @@ func TestStartMGR(t *testing.T) {
 func validateStart(t *testing.T, c *Cluster) {
 
 	for i := 0; i < c.Replicas; i++ {
+		if i == len(mgrNames) {
```

Comment:
Shouldn't this be `i >= len(mgrNames)`?

Reply:
if `i` were incremented somewhere in the loop, `>=` would be necessary, but `==` is sufficient in this case.